docs/specs/parsing.md

```diff
@@ -13,18 +18,75 @@
     - `DateTime`, `DateTimeOffset`
```

Comment:
BTW, as part of the larger cloud framework effort, I think we should consider adding UtcTime type. If we had that, we should consider removing these DateTime structures from the primitives (as far as parsing is concerned).